devopsloft/app/templates/bakery.html

```diff
@@ -2,4 +2,44 @@
 
 {% block body %}
 {% from "includes/_formhelpers.html" import render_field %}
+<div>
+    <p>
+        We have long recognized that there are many jobs for DevOps positions 
+        and on the other hand many candidates without the knowledge required to 
```

Comment:
We have long recognized that there are many vacant openings for DevOps positions 
and many willing talented candidates without the base knowledge required to

Reply:
You hijacked my PR :-) good for you